lib/core/engine.js

```diff
@@ -275,6 +275,16 @@ class Engine {
             );
           }
         })
+        .tap(results => {
+          if (options.chrome && options.chrome.collectConsoleLog) {
+            runner
```

Comment:
This produces a promise, so it needs a return so the outer promise chain waits for it.